apps/frontend/src/emails/templates/moderation/ReportSubmitted.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { Heading, Link as VLink, Text } from '@vue-email/components'
+
+import StyledEmail from '@/emails/shared/StyledEmail.vue'
+</script>
+
+<template>
+	<StyledEmail
+		title="Report submitted successfully"
+		:manual-links="[
+			{ link: 'https://modrinth.com/legal/rules', label: 'Community Guidelines' },
+			{
+				link: 'https://modrinth.com/dashboard/report/{newreport.id}',
+				label: 'View report in dashboard',
+			},
+			{ link: 'https://support.modrinth.com', label: 'Support Portal' },
+		]"
+	>
+		<Heading as="h1" class="mb-2 text-2xl font-bold">Report submitted successfully</Heading>
+
+		<Text class="text-base">Hi {user.name},</Text>
+
+		<Text class="text-base">
+			Thank you for reporting content on Modrinth. We've received your report, {newreport.id}, and
```

Comment:
"We've received your report of {title} and our moderation team will review it shortly."

would move the thank you to after this sentence as well